game/theater/conflicttheater.py

```diff
@@ -270,6 +271,7 @@ def armor_groups(self) -> Iterator[VehicleGroup]:
     def helipads(self) -> Iterator[StaticGroup]:
         for group in self.blue.static_group:
             if group.units[0].type == self.FARP_HELIPAD:
+                print("helooooo")
```

Comment:
Salut!

Reply:
Whoops 😅

Reply:
Resolved in 9ea1edf9dbc8b21c7204ca6a71eab847ddc63522